modules/ROOT/pages/subscriptions/customize-cdc.adoc

```diff
@@ -0,0 +1,39 @@
+[[customize-cdc]]
+= Customize CDC Subscriptions
+:page-aliases: subscriptions/plugins/index.adoc, subscriptions/plugins/amqp.adoc, subscriptions/plugins/single-instance.adoc
+:description: This page describes how  to customize the behaviour of subscriptions.
+
```

Comment:
we should mention what CDC stands for. we don't have an introductory section yet - maybe that would be a good excuse to create one :)